README.md

```diff
@@ -1,117 +1,144 @@
 ![Terrastories](https://www.amazonteam.org/wp-content/uploads/2018/09/logo-1170x164.png)
 
-
 **TerraStories** is an application designed to help communities map and access their own place-based storytelling. The project to develop this application was initiated by the [**Amazon Conservation Team**](http://amazonteam.org) (ACT), an organization who partners with indigenous and other traditional communities in the Amazon rainforest to help them protect their ancestral lands and traditional culture. The application is developed to be entirely open source and offline-compatible, so that it can be used by communities in the most remote locations of the world. It is a Dockerized Rails App that uses [**Mapbox**](https://mapbox.com) to help users locate content geographically on an interactive map. A team is attempting to finish this app at **Ruby for Good 2018**: http://rubyforgood.org/2018
 
 ## Table of Contents
-1. [How To Contribute](#contributing)
 
-2. [Prerequisites](#prerequisites)
+1.  [How to Contribute](#contributing)
+
+2.  [Prerequisites](#prerequisites)
+
+3.  [Setup](#setup)
+
+4.  [Running the Server](#make-it-go)
 
-3. [Setup](#setup)
+5.  [Updating the App](#make-it-go)
 
-4. [Building Map Tiles](#build-the-map-tiles)
+6.  [Development](#development)
 
-5. [Running the server](#make-it-go)
+7.  [Updating the Tileserver Map](#updating-the-tileserver-map)
 
-6. [Updating the tileserver map](#updating-the-tileserver-map)
+## How to Contribute
 
-## Contributing
 We would love for you to contribute to terrastories! We welcome all types of contributions, but any pull requests that address open issues, have test coverage, or are tagged with the next milestone will be prioritized. Please visit our [Contributing Guidelines](CONTRIBUTING.md) for more information.
 
 ## Prerequisites
+
 Install docker. On linux, you may have to install docker-compose separately.
- - https://docs.docker.com/install/
- - https://docs.docker.com/compose/install/
+
+- https://docs.docker.com/install/
+- https://docs.docker.com/compose/install/
 
 ## Setup
-Docker will automatically build images as needed when running `docker-compose up`,
-but to confirm everything builds correctly, run the following and check that the
-output ends with something like this.
+
+This project uses these [GitHub conventions](https://github.com/github/scripts-to-rule-them-all)
+to provide convenient scripts for developers.
+
+On a fresh clone of this repo, run:
+
 ```
-$ docker-compose build
-  ...
-> mariadb uses an image; skipping
-  ...
-> Successfully built 0123456789
-> Successfully tagged terrastories/tilebuilder:latest
-> tileserver uses an image; skipping
-> rails uses an image; skipping
+$ script/setup
```

Comment:
For this and the other script examples ya might want to include the `./` at the beginning of this example for folks that don't have `.` in their `PATH`

E.g.:

```
$ ./script/setup
```